src/main/java/cz/cvut/kbss/termit/persistence/dao/changetracking/ChangeTrackingContextResolver.java

```diff
@@ -54,8 +65,19 @@ private URI resolveTermVocabulary(Term term) {
         } else {
             return em.createNativeQuery("SELECT DISTINCT ?v WHERE { ?t ?inVocabulary ?v . }", URI.class)
                      .setParameter("inVocabulary",
-                                   URI.create(cz.cvut.kbss.termit.util.Vocabulary.s_p_je_pojmem_ze_slovniku))
+                             URI.create(cz.cvut.kbss.termit.util.Vocabulary.s_p_je_pojmem_ze_slovniku))
                      .setParameter("t", term).getSingleResult();
         }
     }
+
+    private URI resolveExistingChangeTrackingContext(URI vocabularyURI) {
+        Optional<URI> vocabularyContext = vocabularies.getVocabularyContext(vocabularyURI);
+        return vocabularyContext.map(uri -> em.createNativeQuery(
+                                                      "SELECT DISTINCT ?ctc WHERE { GRAPH ?vc { ?vc ?hasChangeTrackingContext ?ctc } }", URI.class)
+                                              .setParameter("vc", uri)
+                                              .setParameter("hasChangeTrackingContext",
+                                                      URI.create(
+                                                              cz.cvut.kbss.termit.util.Vocabulary.s_i_ma_kontext_sledovani_zmen))
```

Comment:
Just a minor nitpick - I prefer using the "property" versions of Vocabulary constants over the "individual" versions. So I would replace this with `cz.cvut.kbss.termit.util.Vocabulary.s_p_ma_kontext_sledovani_zmen`.